vendor/github.com/spf13/cobra/command.go

```diff
@@ -967,7 +967,7 @@ func (c *Command) validateRequiredFlags() error {
 func (c *Command) InitDefaultHelpFlag() {
 	c.mergePersistentFlags()
 	if c.Flags().Lookup("help") == nil {
-		usage := "help for "
+		usage := "Help for "
```

Comment:
Don't edit files in `vendor/`, they're external packages we depend on , and edits here will cause weirdness if/when we pull down the latest version of this package.

Reply:
Fixed thanks!